dex/src/main/scala/com/wavesplatform/dex/api/websockets/actors/WsExternalClientHandlerActor.scala

```diff
@@ -128,12 +128,13 @@ object WsHandlerActor {
                   }
 
                 case subscribe: WsOrderBookSubscribe =>
-                  // TODO DEX-700 test for order book that hasn't been created before
                   if (subscribe.depth <= 0) clientRef ! WsError.from(error.RequestArgumentInvalid("depth"), matcherTime)
                   else if (!orderBookSubscriptions.contains(subscribe.key)) {
                     assetPairBuilder.validateAssetPair(subscribe.key).value.onComplete {
-                      case Success(Left(e))  => clientRef ! WsError.from(e, matcherTime)
-                      case Success(Right(_)) => context.self ! Event.AssetPairValidated(subscribe.key)
+                      case Success(Left(e)) => clientRef ! WsError.from(e, matcherTime)
+                      case Success(Right(_)) =>
+                        context.log.debug(s"WsOrderBookSubscribe(k=${subscribe.key}) is successful")
+                        context.self ! Event.AssetPairValidated(subscribe.key)
```

Comment:
Fortunately `context.self` doesn't change in any situations, so we won't lost recipient

Reply:
🎉